### PR TITLE
Update docs for 3.1.0

### PR DESCRIPTION
**Pull request by JakeSCahill**
See commit messages for details.

### CODE REVIEW

File: docs/modules/ROOT/pages/team-and-users.adoc [hunk 1]
```diff
@@ -1,23 +1,85 @@
...
+
+If you've created one or more teams, you can invite users to them and assign those users one of <<user-roles, four roles>>.
+
+. Select an existing team from the dropdown.
++
+image:choose-team.png[Choose an existing team]
+
+. Enter the email address of the team member that you want to add and assign them a <<user-roles, role>> from the dropdown.
++
+image:user-roles.png[A dropdown menu with a list of roles]
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/team-and-users.adoc:73]: Image is not found

Comment (huseyinbabal) [inline docs/modules/ROOT/pages/team-and-users.adoc:73]: Image is not found
Screen Shot 2021-09-02 at 15 20 51

File: docs/modules/ROOT/pages/team-and-users.adoc [hunk 2]
```diff
@@ -1,23 +1,85 @@
...
+
+. Click *Invite*.
++
+The user will receive an email invitation. When the user accepts the invitation, the user's status will change to 'ACTIVE'.
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/team-and-users.adoc:77]: Better to have bold **ACTIVE** instead enclosed by single quote?

File: docs/modules/ROOT/pages/payment-methods.adoc [hunk 3]
```diff
@@ -1,5 +1,13 @@
-= Payment Methods
+= Managing Payment Methods
 
-Hazelcast Cloud works with Stripe Inc for safely processing payments. We never keep any credit card data in our database. You can add your credit card details under *Account* > *Payment Methods* menu as below:
+Hazelcast Cloud works with Stripe Inc for safely processing payments. We never keep any credit card data in our database.
+
+== Prerequisites
+
+To manage payment methods for a team, you must have either an admin or finance role. See xref:teams-and-users.adoc[].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/payment-methods.adoc:7]: I see `.adoc` url in the referred link text, is that valid usage? We dont need to show adoc extension to end user
Screen Shot 2021-09-02 at 15 24 19

File: docs/modules/ROOT/pages/credits.adoc [hunk 4]
```diff
@@ -1,9 +1,15 @@
...
+= Managing Credits
 
-If you have a promo code then you can use it to redeem credit. As long as your credit balance is positive, any usage will be deducted from your credit balance. When you are out of credit, and if you do not have any other payment method defined, then your clusters will be stopped. 
+If you have a promo code then you can use it to redeem credit. As long as your credit balance is positive, any usage will be deducted from your credit balance. When you are out of credit, and if you do not have any other payment method defined, then your clusters will be stopped.
 
-You can see the remaining credit balance under *Account* > *Billing & Payments* page.
+== Prerequisites
 
-You can redeem credit under Account > Credits page, as shown below:  
+To manage credits for a team, you must have either an admin or finance role. See xref:teams-and-users.adoc[].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/credits.adoc:7]: There is adoc extension in link text
Screen Shot 2021-09-02 at 15 26 21

File: docs/modules/ROOT/pages/developer.adoc [hunk 5]
```diff
@@ -1,27 +1,29 @@
...
 :url-cloud-developers: https://cloud.hazelcast.com/settings/developer
 :url-cloud-api: https://cloud.hazelcast.com/v1/api/explorer
 
-The Hazelcast Cloud allows you to manage your resources and clusters via link:{url-cloud-api}[API], link:{url-github-go-sdk}[SDK], and link:{url-github-cloud-cli}[CLI]. You can get further details about them from xref:api-reference.adoc[API Reference].
+Hazelcast Cloud allows you to manage your resources and clusters via an link:{url-cloud-api}[API], an link:{url-github-go-sdk}[SDK], and a link:{url-github-cloud-cli}[CLI]. To use these applications, you need an API key and an API secret, which can be created from link:{url-cloud-developers}[Developers] tab in your account or team settings. 
 
-In order to use these applications, you need to provide API Key and API Secret. This API Key and API Secret can be created from link:{url-cloud-developers}[Developers] tab in Settings as shown below. 
+== Prerequisites
+
+To manage API keys for a team's cluster, you must have either an admin or developer role. See xref:teams-and-users.adoc[].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/developer.adoc:11]: adoc extension in link text
Screen Shot 2021-09-02 at 15 26 21

File: docs/modules/ROOT/pages/azure-vnet-peering.adoc [hunk 6]
```diff
@@ -1,31 +1,22 @@
...
-- Assign Role
-- Create, Read, Update on the Virtual Networks that you will create peering.
-- Peer Action on the Virtual Networks that you will create peering.
-+
-TIP: You can check further details about Azure CLI and download it from link:{url-azure-cli}[Azure Command-Line Interface (CLI) Documentation].
+link:{url-azure-cli}[Download the Azure CLI] and make sure you have the link:{url-azure-permissions}[correct permissions].
 
-=== 2) Create Peering
-After you successfully complete prerequisites, you can easily create Azure vNet Peering using Hazelcast Cloud CLI by running the following command.
+To create a peering connection on a team cluster, you must have either an admin or developer role. See xref:teams-and-users.adoc[].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/azure-vnet-peering.adoc:14]: adoc extension in link text

File: docs/modules/ROOT/pages/backup-and-restore.adoc [hunk 7]
```diff
@@ -7,9 +7,13 @@ NOTE: If the hot-backup feature is enabled, your cluster will create a backup au
 
 WARNING: You can keep at most 10 backups. The oldest backup will be deleted automatically after the 10th backup.
 
+== Prerequisites
+
+To back up or restore clusters in a team, you must have either an admin or developer role. See xref:teams-and-users.adoc[].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/backup-and-restore.adoc:12]: adoc extension in link text

File: docs/modules/ROOT/pages/create-enterprise-cluster.adoc [hunk 8]
```diff
@@ -1,26 +1,81 @@
...
-* *Memory*: The cluster creation mechanism gives you a total memory of 7GB for each Hazelcast IMDG member in your cluster. You can change the native and heap memory capacity using the *Change* button in the field. You can see a summary of your memory distribution as you change the sizes at the very bottom of the page under the "Summary" heading.
-* *Backup/Snapshot*: Enable or disable Hazelcast's Hot Backup feature. If you enable it, your cluster will be backed-up daily. In addition, you can create snapshots and you will be able to restore your cluster from a previous snapshot or backup later. 
-* *Encryption*: Enable or disable the TLS communication between the clients and members.
-* *Custom Configuration*: You can specify your own configurations for Hazelcast IMDG data structures in your cluster. If you desire so, click on the *Add* button in this field and configure the data structures using the appeared form which is shown below:
+To create a cluster for a team, you must have either an admin or developer role. See xref:teams-and-users.adoc[].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/create-enterprise-cluster.adoc:11]: adoc extension in link text

File: docs/modules/ROOT/pages/create-standard-cluster.adoc [hunk 9]
```diff
@@ -0,0 +1,57 @@
+= Creating a Standard Cluster
+:description: Standard clusters are ideal for small or medium applications that need the flexibility to automatically scale up or down depending on the amount of used memory. 
+:page-aliases: create-starter-cluster.adoc, starter-clusters.adoc
+
+{description}
+
+== Prerequisites
+
+To create a cluster for a team, you must have either an admin or developer role. See xref:teams-and-users.adoc[].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/create-standard-cluster.adoc:9]: adoc extension in link text

File: docs/modules/ROOT/pages/custom-classes-upload.adoc [hunk 10]
```diff
@@ -1,31 +1,31 @@
...
 
-If you need your uploaded code to be granted any additional permissions, please create a support ticket for it.
-====
+Your files must be in either `zip` or `jar` format and the maximum file size is 500 MB.
 
-== Uploading Custom Classes
+Uploaded classes are restricted by a limited set of JSP permissions. If you need your uploaded code to be granted any additional permissions, link:https://hazelcast.zendesk.com/hc/en-us/signin?return_to=https%3A%2F%2Fhazelcast.zendesk.com%2Fhc%2Fen-us%2Frequests%2Fnew[create a support ticket] for it.
 
-To upload JAR files with custom classes, you need to click on the *Upload Custom Classes* button in the cluster details page:
+To upload custom classes onto a team's cluster, you must have either an admin or developer role. See xref:teams-and-users.adoc[].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/custom-classes-upload.adoc:22]: adoc extension in link text

File: docs/modules/ROOT/pages/gcp-vpc-peering.adoc [hunk 11]
```diff
@@ -7,19 +7,18 @@
...
+There are 2 ways of creating a GCP VPC Peering connection:
 
-== Create Peering
+- Use the CLI (recommended)
+- Create a manual connection
 
-There are 2 ways of creating a GCP VPC Peering. 
-- First Option (Easy and Recommended) 
-- Second Option (Manual)
+To create a peering connection on a team cluster, you must have either an admin or developer role. See xref:teams-and-users.adoc[].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/gcp-vpc-peering.adoc:15]: adoc extension in link text

File: docs/modules/ROOT/pages/hazelcast-version.adoc [hunk 12]
```diff
@@ -1,20 +1,18 @@
...
+{description}
 
-== When Creating a Cluster
+== Prerequisites
 
-Simply choose the version in the Hazelcast Version dropdown menu of the cluster creation form:
+Your cluster must be on the link:{page-plans}[Enterprise plan] to use this feature.
 
-image:hazelcast-version-selection.png[Selecting a version of Hazelcast while creating a new cluster]
+To update the version of a team's cluster, you must have either an admin or developer role. See xref:teams-and-users.adoc[].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/hazelcast-version.adoc:10]: adoc extension in link text

File: docs/modules/ROOT/pages/ip-white-list.adoc [hunk 13]
```diff
@@ -1,17 +1,30 @@
...
+{description}
 
-== How To Enable
+== Prerequisites
 
-IP whitelisting can be enabled when creating the cluster, as shown below:
+If you're on the Enterprise plan, you can use IP whitelisting only if you turned on public access while xref:create-enterprise-cluster.adoc[creating the cluster]. Otherwise, your cluster is private and you need to xref:connect-to-private-cluster.adoc[set up peering to allow clients to connect to it].
 
-image:enable-ip-whitelist.png[Enable IP whitelist and add IP addresses to the list]
+To add IP addresses to the whitelist of a team's cluster, you must have either an admin or developer role. See xref:teams-and-users.adoc[].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/ip-white-list.adoc:10]: adoc extension in link text

File: docs/modules/ROOT/pages/manage-the-cluster.adoc [hunk 14]
```diff
@@ -1,45 +1,26 @@
...
+:description: After creating a cluster, you can either stop it to save the configuration but stop charges accumulating or you can delete it all together.
 
-== Scaling Cluster
+{description}
 
-Once you have a running cluster, you can scale up or down depending on your requirements. To do so, simply click on the *Manage Memory* button on the Cluster page and adjust the memory using the slider, as shown below.
+== Prerequisites
 
-image:manage-memory.png[Slide the bar to choose how much memory to allocate the cluster]
+To stop or delete clusters in a team, you must have either an admin or developer role. See xref:teams-and-users.adoc[].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/manage-the-cluster.adoc:8]: adoc extension in link text

File: docs/modules/ROOT/pages/maploader-and-mapstore.adoc [hunk 15]
```diff
@@ -1,6 +1,12 @@
 = MapLoader and MapStore
 :url-code-sample-mapstore: https://github.com/hazelcast/hazelcast-cloud-code-samples/tree/master/mapstore
 
+== Prerequisites
+
+Your cluster must be on the link:{page-plans}[Enterprise plan] to use this feature.
+
+To use this feature, you must have either an admin or developer role. See xref:teams-and-users.adoc[].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/maploader-and-mapstore.adoc:8]: adoc extension in link text

File: docs/modules/ROOT/nav.adoc [hunk 16]
```diff
@@ -1,40 +1,55 @@
...
-*** xref:backup-and-restore.adoc[]
+.Account Management
+** xref:account-security.adoc[]
+** xref:team-and-users.adoc[]
+** xref:payment-methods.adoc[]
+** xref:credits.adoc[]
+** xref:developer.adoc[]
+
+.Create Clusters
+** link:{page-plans}[Standard vs Enterprise^]
```
Comment (huseyinbabal) [inline docs/modules/ROOT/nav.adoc:16]: Dead link

**Comment by huseyinbabal** [comment]
@JakeSCahill are you planning to change screenshots for example client code sections? I see they are old designs https://deploy-preview-32--xenodochial-spence-280948.netlify.app/cloud/net-client.html#quick-setup

File: docs/modules/ROOT/pages/payment-methods.adoc (hunk 3 above)
Comment (JakeSCahill) [reply docs/modules/ROOT/pages/payment-methods.adoc:7]: All of these .adoc links were broken. They are fixed now 😄

File: docs/modules/ROOT/pages/java-client.adoc [hunk 17]
```diff
@@ -269,4 +269,4 @@ HazelcastInstance client = HazelcastClient.newHazelcastClient();
 
 == More Configuration Options
 
-Please refer to the xref:imdg:clients:java.adoc#configuring-java-client[Hazelcast IMDG Reference Manual] for further configuration options.
\ No newline at end of file
+Please refer to the xref:hazelcast:clients:java.adoc#configuring-java-client[Java client documentation] for further configuration options.
```
[the remark this answers is not in the document]
Comment (JakeSCahill) [reply docs/modules/ROOT/pages/java-client.adoc:272]: These will work when we merge.

**Comment by JakeSCahill** [comment]
> @JakeSCahill are you planning to change screenshots for example client code sections? I see they are old designs https://deploy-preview-32--xenodochial-spence-280948.netlify.app/cloud/net-client.html#quick-setup

Yes. Are all the pages finished now? We were waiting until development had finished to check the remaining screenshots.

**Comment by huseyinbabal** [comment]
@JakeSCahill very general comment, we state a link as external reference but open it same window is this expected? e.g. 
Screen Shot 2021-09-02 at 16 31 17

File: docs/modules/ROOT/pages/map-configurations.adoc [hunk 18]
```diff
@@ -5,7 +5,7 @@ image:map-configuration.png[Map configuration]
 == Map Name
 
 In Hazelcast, you can create custom map configurations associated with the names of maps. The maps that do not have any configuration defined use the default configuration. If you want to set a configuration that is valid for all maps, you can name your configuration as `default`. A user-defined `default` configuration applies to every map that does not have a specific custom map configuration defined with the map's name. 
-You can also use wildcards to associate your configuration with multiple map names. See the xref:imdg:configuration:using-wildcards.adoc[IMDG documentation] for more information about wildcards.
+You can also use wildcards to associate your configuration with multiple map names. See the xref:hazelcast:configuration:using-wildcards.adoc[configuration documentation] for more information about wildcards.
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/map-configurations.adoc:8]: dead link

File: docs/modules/ROOT/pages/jcache.adoc [hunk 19]
```diff
@@ -3,7 +3,7 @@
 == JCache Name
 
 In Hazelcast, you can create custom jcache configurations associated with the names of jcaches. The jcaches that do not have any configuration defined use the default configuration. If you want to set a configuration that is valid for all jcaches, you can name your configuration as `default`. A user-defined `default` configuration applies to every jcache that does not have a specific custom jcache configuration defined with the jcache's name. 
-You can also use wildcards to associate your configuration with multiple jcache names. See the xref:imdg:configuration:using-wildcards.adoc[IMDG documentation] for more information about wildcards.
+You can also use wildcards to associate your configuration with multiple jcache names. See the xref:hazelcast:configuration:using-wildcards.adoc[configuration documentation] for more information about wildcards.
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/jcache.adoc:6]: dead link

File: docs/modules/ROOT/pages/queue.adoc [hunk 20]
```diff
@@ -3,7 +3,7 @@
 == Queue Name
 
 In Hazelcast, you can create custom queue configurations associated with the names of queues. The queues that do not have any configuration defined use the default configuration. If you want to set a configuration that is valid for all queues, you can name your configuration as `default`. A user-defined `default` configuration applies to every queue that does not have a specific custom queue configuration defined with the queue's name. 
-You can also use wildcards to associate your configuration with multiple queue names. See the xref:imdg:configuration:using-wildcards.adoc[IMDG documentation] for more information about wildcards.
+You can also use wildcards to associate your configuration with multiple queue names. See the xref:hazelcast:configuration:using-wildcards.adoc[configuration documentation] for more information about wildcards.
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/queue.adoc:6]: dead link

File: docs/modules/ROOT/pages/replicated-map.adoc [hunk 21]
```diff
@@ -3,11 +3,11 @@
 == Replicated Map Name
 
 In Hazelcast, you can create custom replicated map configurations associated with the names of replicated maps. The replicated maps that do not have any configuration defined use the default configuration. If you want to set a configuration that is valid for all replicated maps, you can name your configuration as `default`. A user-defined `default` configuration applies to every replicated map that does not have a specific custom replicated map configuration defined with the replicated map's name. 
-You can also use wildcards to associate your configuration with multiple replicated map names. See the xref:imdg:configuration:using-wildcards.adoc[IMDG documentation] for more information about wildcards.
+You can also use wildcards to associate your configuration with multiple replicated map names. See the xref:hazelcast:configuration:using-wildcards.adoc[configuration documentation] for more information about wildcards.
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/replicated-map.adoc:6]: dead link

File: docs/modules/ROOT/pages/replicated-map.adoc [hunk 22]
```diff
@@ -3,11 +3,11 @@
...
 
 == In Memory Format
 
-Internal storage format. See the xref:imdg:data-structures:replicated-map.adoc#in-memory-format-on-replicated-map[In-Memory Format section]. Its default value is `OBJECT`.
+Internal storage format. See the xref:hazelcast:data-structures:replicated-map.adoc#in-memory-format-on-replicated-map[In-Memory Format section]. Its default value is `OBJECT`.
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/replicated-map.adoc:10]: dead link

File: docs/modules/ROOT/pages/set.adoc [hunk 23]
```diff
@@ -8,7 +8,7 @@ WARNING: Since the whole set lives on a single partition, storing a large amount
 == Set Name
 
 In Hazelcast, you can create custom set configurations associated with the names of the Set. The Set that does not have any configuration defined will use the default configuration. If you want to set a configuration that is valid for all sets, you can name your configuration as `default`. A user-defined `default` configuration applies to every set that does not have a specific custom set configuration defined with the set's name. 
-You can also use wildcards to associate your configuration with multiple set names. See the xref:imdg:configuration:using-wildcards.adoc[IMDG documentation] for more information about wildcards.
+You can also use wildcards to associate your configuration with multiple set names. See the xref:hazelcast:configuration:using-wildcards.adoc[configuration documentation] for more information about wildcards.
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/set.adoc:11]: dead link

File: docs/modules/ROOT/pages/list.adoc [hunk 24]
```diff
@@ -5,7 +5,7 @@ Hazelcast List (*IList*) is similar to Hazelcast Set, but it also allows duplica
 == List Name
 
 In Hazelcast, you can create custom list configurations associated with the names of list. The list that do not have any configuration defined use the default configuration. If you want to set a configuration that is valid for all lists, you can name your configuration as `default`. A user-defined `default` configuration applies to every list that does not have a specific custom list configuration defined with the list's name. 
-You can also use wildcards to associate your configuration with multiple list names. See the xref:imdg:configuration:using-wildcards.adoc[IMDG documentation] for more information about wildcards.
+You can also use wildcards to associate your configuration with multiple list names. See the xref:hazelcast:configuration:using-wildcards.adoc[configuration documentation] for more information about wildcards.
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/list.adoc:8]: dead link

File: docs/modules/ROOT/pages/topic.adoc [hunk 25]
```diff
@@ -5,7 +5,7 @@ Hazelcast provides a distribution mechanism for publishing messages that are del
 == Topic Name
 
 In Hazelcast, you can create custom topic configurations associated with the names of the topic. The topic that does not have any configuration defined use the default configuration. If you want to set a configuration that is valid for all topics, you can name your configuration as `default`. A user-defined `default` configuration applies to every topic that does not have a specific custom topic configuration defined with the topic's name. 
-You can also use wildcards to associate your configuration with multiple topic names. See the xref:imdg:configuration:using-wildcards.adoc[IMDG documentation] for more information about wildcards.
+You can also use wildcards to associate your configuration with multiple topic names. See the xref:hazelcast:configuration:using-wildcards.adoc[configuration documentation] for more information about wildcards.
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/topic.adoc:8]: dead link

File: docs/modules/ROOT/pages/failover-clients-with-hazelcast-cloud.adoc [hunk 26]
```diff
@@ -4,11 +4,9 @@ Failover is an important feature of the mission-critical systems that rely on ne
 
 In this tutorial, we will create a setup with Failover Configuration using multiple Hazelcast Cloud Clusters.
 
-== Create Multiple Clusters
+== Before you Begin
 
-In order to achieve failover configuration, you need to create multiple clusters on Hazelcast Cloud. So, we can setup which clusters will get into action when there is a problem with one of them.
-
-TIP: You can check how to create Starter and Enterprise Clusters from xref:create-enterprise-cluster.adoc[Creating Enterprise Cluster] and xref:create-starter-cluster.adoc [Creating Starter Cluster].
+In order to achieve failover configuration, you need to create multiple clusters on Hazelcast Cloud. So, we can set up which clusters will get into action when there is a problem with one of them. See xref:create-enterprise-cluster.adoc[] or xref:create-standard-cluster.adoc [].
```
Comment (huseyinbabal) [inline docs/modules/ROOT/pages/failover-clients-with-hazelcast-cloud.adoc:9]: adoc extension in link text and it is dead

**Review by huseyinbabal — APPROVED** [review]
LGTM after you revisit addressed sections, well done @JakeSCahill !

**Comment by JakeSCahill** [comment]
> @JakeSCahill very general comment, we state a link as external reference but open it same window is this expected? e.g.
> Screen Shot 2021-09-02 at 16 31 17

Good point -  I think it should open a new window I'll open an issue on our UI repo 😄

**Comment by huseyinbabal** [comment]
Fixes #HZC-1618

**Pull request merged**